montie/session_03/mailroom.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+donors = ["Don Rickles","Mitch Hedberg","Dean Martin","Joan Rivers","Charles Rocket"]
+donation_amount = ["50:20:100","70","491:36","10:50:71","87:92"]
```

Comment:
Interesting way to approach this keeping the stings and using `:` as a delimiter.